steps/property-inspection/repairs.tsx

```diff
@@ -0,0 +1,134 @@
+import { FieldsetLegend, Paragraph } from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentValue,
+  ComponentWrapper,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+
+import { ImageInput } from "../../components/ImageInput";
+import { makeSubmit } from "../../components/makeSubmit";
+import { RadioButtons } from "../../components/RadioButtons";
+import { TextArea } from "../../components/TextArea";
+
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import processRef from "../../storage/processRef";
+
+import PageSlugs, { urlObjectForSlug } from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step = {
+  title: PageTitles.Repairs,
+  heading: "New repairs",
+  step: {
+    slug: PageSlugs.Repairs,
+    nextSlug: PageSlugs.StoringMaterials,
+    submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
+      makeSubmit({
+        url: urlObjectForSlug(nextSlug),
+        value: "Save and continue"
+      }),
+    componentWrappers: [
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "paragraph-1",
+          Component: Paragraph,
+          props: {
+            children: `Residents can contact the Repairs Contact Centre: 020 8356 3691
+            or [Report a repair] online (online only, opens in a new tab).`
+          }
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "has-repair",
+          Component: RadioButtons,
+          props: {
+            name: "has-repair",
+            legend: (
+              <FieldsetLegend>Are there any new repair queries?</FieldsetLegend>
+            ) as React.ReactNode,
+            radios: [
+              {
+                label: "Yes",
+                value: "yes"
+              },
+              {
+                label: "No",
+                value: "no"
+              }
+            ]
+          },
+          defaultValue: "",
+          emptyValue: "",
+          databaseMap: new ComponentDatabaseMap<
+            ProcessDatabaseSchema,
+            "property"
+          >({
+            storeName: "property",
+            key: processRef,
+            property: ["repair", "hasRepair"]
+          })
+        })
+      ),
+      ComponentWrapper.wrapDynamic(
+        new DynamicComponent({
+          key: "repair-images",
+          Component: ImageInput,
+          props: {
+            label: "Take photo of repair needed",
+            name: "repair-images",
+            hintText: "" as string | null | undefined,
+            maxCount: 1 as number | null | undefined
```

Comment:
Let's make this much larger. 10? See if Gill has a reckon on how many, and note the limit like we do for other questions.

Reply:
Wasn't sure about this as Gill wanted each new picture to be associated with a new note. Damages is max count 5 and garden is 3. happy to do 10 if you think that's a good solution for now.

Reply:
I'd like to leave this in a state where if we don't get to doing multiple repairs (which is non-trivial), the housing officer can still document what they need to.